config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  patch "/guests/:id", to: "guest_rooms#create"
```

Comment:
reference my comment on your guest show page to see how I'd update this route ;)